src/lib/components/column-profile/ContextButton.svelte

```diff
@@ -20,6 +21,18 @@
   <button
     on:click
     {id}
+    on:focus={() => {
```

Comment:
Would it be possible to get this as a plugin? I would think this would be needed in other places later on.

Reply:
was just thinking that. Going to make an [action](https://svelte.dev/tutorial/actions) for it.